manifests/plugin/dns_route53.pp

```diff
@@ -6,17 +6,19 @@
 # @param propagation_seconds Number of seconds to wait for the DNS server to propagate the DNS-01 challenge.
 # @param manage_package Manage the plugin package.
 # @param package_name The name of the package to install when $manage_package is true.
+# @param package_ensure The value passed to `ensure` when installing the plugin package.
 #
 class letsencrypt::plugin::dns_route53 (
   String[1] $package_name,
   Integer $propagation_seconds     = 10,
   Boolean $manage_package          = true,
+  String $package_ensure           = $letsencrypt::package_ensure,
```

Comment:
As long as we `require letsencrypt`, we can use `$letsencrypt::package_ensure` without the need to add parameters.  Unless there are use cases for setting an explicit different value, I would recommend not adding a new parameter.